src/components/CreateBeer.js

```diff
@@ -0,0 +1,57 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+function CreateBeer() {
+
+    const [name, setName] = useState("");
+    const [tagline, setTagline] = useState("");
+    const [description, setDescription] = useState("");
+    const [first_brewed, setFirstBrewed] = useState("");
+    const [brewers_tips, setBrewersTips] = useState("");
+    const [attenuation_level, setAttenuation] = useState();
+    const [contributed_by, setContributed] = useState("");
+    const [newBeer, setNewBeer] = useState({});
+    let navigate = useNavigate();
+
+    useEffect(() => {
+        axios
+            .post('https://ih-beers-api2.herokuapp.com/beers/new', newBeer)
+            .then(() => {
+                return (
+                    navigate("/beers", { replace: true })
+                )
+            })
+            .catch(function (error) {
+                console.log(error);
+            });
+    }, [newBeer])
+
+    const handleCreate = (e) => {
+        e.preventDefault();
+        setNewBeer(preValue => {
+            const obj = { name, tagline, description, first_brewed, brewers_tips, attenuation_level, contributed_by };
+            return obj
+        });
+    }
```

Comment:
You have the state for every input in your form -> and you update state onChange.
So no need to have additional state to hold the newBeer object. 

Your handleCreate method could look smth like this:
- you create object using the states you have and send it to db

```
    const handleCreate = (e) => {
        e.preventDefault();
        const newBeer = { name, tagline, description, first_brewed, brewers_tips, attenuation_level, contributed_by };

        axios
        .post('https://ih-beers-api2.herokuapp.com/beers/new', newBeer)
        .then(() => {
            return (
                navigate("/beers", { replace: true })
            )
        })
        .catch( (error)=> {
            console.log(error);
        });
    }
```